### ISSUE TITLE
Failed Go Mod Tidy

### ISSUE THREAD

**Issue by KiddoV** [spf13/viper#1292]
### Preflight Checklist

- [X] I have searched the [issue tracker](https://www.github.com/spf13/viper/issues) for an issue that matches the one I want to file, without success.
- [X] I am not looking for support or already pursued the available [support channels](https://github.com/spf13/viper/issues/new/choose) without success.
- [X] I have checked the [troubleshooting guide](https://github.com/spf13/viper/blob/master/TROUBLESHOOTING.md) for my problem, without success.

### Viper Version

1.10.1

### Go Version

go version go1.17.6 windows/amd64

### Config Source

Files

### Format

JSON

### Expected Behavior

``go mod tidy`` act normally!

### Actual Behavior

``go mod tidy``
Output:
```cmd
appmodule imports
        github.com/spf13/viper imports
        github.com/spf13/viper/internal/encoding/yaml imports
        gopkg.in/yaml.v2 tested by
        gopkg.in/yaml.v2.test imports
        gopkg.in/check.v1 loaded from gopkg.in/check.v1@v0.0.0-20161208181325-20d25e280405,
        but go 1.16 would select v1.0.0-20201130134442-10cb98267c6c

To upgrade to the versions selected by go 1.16:
        go mod tidy -go=1.16 && go mod tidy -go=1.17
If reproducibility with go 1.16 is not needed:
        go mod tidy -compat=1.17
For other options, see:
        https://golang.org/doc/modules/pruning
```

### Additional Information

This is my first time using Viper. Thanks for a great package!

**Comment by KiddoV** [comment]
Problem solve! False call!

**Comment by s-titov** [comment]
@KiddoV I have same problem, how did you solve it?

**Comment by KiddoV** [comment]
@s-titov Yes, if u use the normal terminal do ``go mod tidy -go 1.16`` without the ``=`` and then ``go mod tidy -go 1.17``
